tests/integration/features/callapi/recording.feature

```diff
@@ -116,3 +116,17 @@ Feature: callapi/recording
     And the response error matches with "call"
     Then user "participant1" starts "audio" recording in room "room1" with 400 (v1)
     And the response error matches with "call"
+
+  Scenario: Store recording
+    When the following "spreed" app config is set
+      | signaling_dev | yes |
+    And user "participant1" creates room "room1" (v4)
+      | roomType | 2 |
+      | roomName | room1 |
+    And user "participant1" joins room "room1" with 200 (v4)
+    And user "participant1" joins call "room1" with 200 (v4)
+    And user "participant1" starts "video" recording in room "room1" with 200 (v1)
+    And user "participant1" stops recording in room "room1" with 200 (v1)
+    Then user "participant1" store recording file "/img/join_call.ogg" in room "room1" with 200 (v1)
```

Comment:
@nickvergessen how will we identify if the file was from a recording? Or we don't will do this?
At this time the previous steps aren't necessary because we haven't any parameter to identify if the file are from the previous recording.

Reply:
> At this time the previous steps aren't necessary because we haven't any parameter to identify if the file are from the previous recording.

I would say we don't care. If the request was authenticated with the SIP credentials that's enough